src/flags.ts

```diff
@@ -228,67 +235,56 @@ export const STARTING_CSV = string({
-                    By default this flag is enabled.
-                    This parameter is used only when the installer is 'olm'.`,
+                    By default this flag is enabled.`,
   allowNo: true,
   default: true,
-  exclusive: [STARTING_CSV_FLAG],
 })
 
 export const OLM_CHANNEL_FLAG = 'olm-channel'
 export const OLM_CHANNEL = string({
   description: `Olm channel to install ${EclipseChe.PRODUCT_NAME}, f.e. stable.
-                    If options was not set, will be used default version for package manifest.
-                    This parameter is used only when the installer is the 'olm'.`,
+                    If options was not set, will be used default version for package manifest.`,
 })
 
 export const PACKAGE_MANIFEST_FLAG = 'package-manifest-name'
 export const PACKAGE_MANIFEST = string({
-  description: `Package manifest name to subscribe to ${EclipseChe.PRODUCT_NAME} OLM package manifest.
-                    This parameter is used only when the installer is the 'olm'.`,
+  description: `Package manifest name to subscribe to ${EclipseChe.PRODUCT_NAME} OLM package manifest.`,
 })
 
 export const CATALOG_SOURCE_NAMESPACE_FLAG = 'catalog-source-namespace'
 export const CATALOG_SOURCE_NAME_FLAG = 'catalog-source-name'
-
+export const CATALOG_SOURCE_IMAGE_FLAG = 'catalog-source-image'
 export const CATALOG_SOURCE_YAML_FLAG = 'catalog-source-yaml'
+
 export const CATALOG_SOURCE_YAML = string({
   description: `Path to a yaml file that describes custom catalog source for installation ${EclipseChe.PRODUCT_NAME} operator.
                     Catalog source will be applied to the namespace with ${EclipseChe.PRODUCT_NAME} operator.
-                    Also you need define 'olm-channel' name and 'package-manifest-name'.
-                    This parameter is used only when the installer is the 'olm'.`,
-  exclusive: [CATALOG_SOURCE_NAME_FLAG, CATALOG_SOURCE_NAMESPACE_FLAG],
-  dependsOn: [OLM_CHANNEL_FLAG, PACKAGE_MANIFEST_FLAG],
+                    Also you need define 'olm-channel' name and 'package-manifest-name'.`,
+  exclusive: [CATALOG_SOURCE_NAME_FLAG, CATALOG_SOURCE_NAMESPACE_FLAG, CATALOG_SOURCE_IMAGE_FLAG],
 })
 
 export const CATALOG_SOURCE_NAMESPACE = string({
-  description: `Namespace for OLM catalog source to install ${EclipseChe.PRODUCT_NAME} operator.
-                    This parameter is used only when the installer is the 'olm'.`,
+  description: `Namespace for OLM catalog source to install ${EclipseChe.PRODUCT_NAME} operator.`,
   dependsOn: [CATALOG_SOURCE_NAME_FLAG],
-  exclusive: [CATALOG_SOURCE_YAML_FLAG],
+  exclusive: [CATALOG_SOURCE_YAML_FLAG, CATALOG_SOURCE_IMAGE_FLAG],
 })
 
 export const CATALOG_SOURCE_NAME =  string({
-  description: `OLM catalog source to install ${EclipseChe.PRODUCT_NAME} operator.
-                    This parameter is used only when the installer is the 'olm'.`,
+  description: `OLM catalog source name to install ${EclipseChe.PRODUCT_NAME} operator.`,
```

Comment:
Does this have a default value, and is therefore not required? For example if I pass in `--catalog-source-image=quay.io/devspaces/iib:3.6.0.RC-04-27-v4.11-x86_64` could the catalog source name be generated as `iib-3-6-0-rc-04-27-v4-11-x86-64-fast` ... similar to https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/product/installCatalogSourceFromIIB.sh#L262-L276 ? 

```suggestion
  description: `Name of the OLM catalog source or index bundle (IIB) from which to install ${EclipseChe.PRODUCT_NAME} operator.`,
```

Reply:
It is `devspaces-fast`
https://github.com/redhat-developer/devspaces-chectl/blob/devspaces-3-rhel-8/build/scripts/sync.sh#L104

Reply:
But I agree it's better to generate a name.